static/js/timerender.js

```diff
@@ -40,6 +40,39 @@ exports.render_now = function (time) {
     return [time.toString("MMM\xa0dd"), false];
 };
 
+// Current date is passed as an argument for unit testing
+exports.last_seen_status_from_date = function (last_active_date, current_date) {
+    if (typeof  current_date === 'undefined') {
+         current_date = new XDate();
+    }
+    var start_of_today = set_to_start_of_day(current_date.clone());
+    var start_of_other_day = set_to_start_of_day(last_active_date.clone());
+    var days_old = Math.round(start_of_other_day.diffDays(start_of_today));
+    if (days_old === 0) {
+        var hours_old = last_active_date.diffHours(current_date);
+        if (hours_old < 1) {
+            var minutes_old = Math.round(last_active_date.diffMinutes(current_date));
+            if (minutes_old === 0) {
+                return i18n.t("Last seen just now");
+            }
+            if (minutes_old === 1) {
+                return i18n.t("Last seen a minute ago");
+            }
+            return i18n.t("Last seen " + minutes_old +" minutes ago");
```

Comment:
You can't call `i18n.t` on computed strings.  You need to something like this:

~~~ js
var context = {minutes_ago: minutes_old};
return i18n.t("Last seen __minutes_ago__ minutes ago", context);